src/app/server/DefaultArlStorage.h

```diff
@@ -0,0 +1,37 @@
+/*
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app/server/ArlStorage.h>
+
+namespace chip {
+namespace app {
+
+class DefaultArlStorage : public ArlStorage
```

Comment:
Please call this `ExampleArlStorage`. Calling it default assumes many implementers would use it as-is, but ARL handling is 100% product specific due to how overrides can be requested, so this should be an example only.